packages/matchbox/src/components/ThemeProvider/ThemeProvider.js

```diff
@@ -18,22 +39,25 @@ const GlobalStyle = createGlobalStyle`
  *
  * @prop theme Overrides matchbox's theme
  */
-
 function Theme(props) {
   return (
-    <ThemeProvider theme={{ ...theme, ...props.theme }}>
-      {/* 
-        FE-913:`skipGlobalStyles` is to exclude global styles from unit tests.
-        The custom assertion `toHaveAttributeValue` fails with this turned on.
-        We should either move away from using this assertion (or even enzyme),
-        or try to resolve the error
-      */}
-      {!props.skipGlobalStyles && <GlobalStyle />}
-      {props.children}
-    </ThemeProvider>
+    <Manager>
+      <ThemeProvider theme={{ ...theme, ...props.theme }}>
+        {/*
+          FE-913:`skipGlobalStyles` is to exclude global styles from unit tests.
+          The custom assertion `toHaveAttributeValue` fails with this turned on.
+          We should either move away from using this assertion (or even enzyme),
+          or try to resolve the error
+        */}
```

Comment:
I think the time to move to React Testing Library is at hand!

Reply:
(I know this comment wasn't introduced with this ticket, but if you're ever interested in swapping let me know - I can walk you through it)

Reply:
Heh, yea I kind of regret not making the switch before we started all the component rework